### PR TITLE
more CLI tweaks

### PR DESCRIPTION
**Pull request by ryanpetrello**

### CODE REVIEW

File: awxkit/awxkit/cli/resource.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
 CONTROL_RESOURCES = ['ping', 'config', 'me', 'metrics']
 
 DEPRECATED_RESOURCES = {
+    'ad_hoc_commands': 'ad_hoc',
```
Comment (ryanpetrello) [inline awxkit/awxkit/cli/resource.py:14]: @wenottingham and I just missed this when adding `tower-cli` alias support cc @squid

**Review by chrismeyersfsu — APPROVED** [review]

File: awxkit/awxkit/cli/format.py [hunk 2]
```diff
@@ -136,32 +137,49 @@ def format_yaml(output, fmt):
...
-            dict(
-                (col, record.get(col, ''))
-                for col in column_names
-            )
-            for record in output
-        ],
-        headers='keys',
-        tablefmt='rst'
-    )
+    if fmt == '*' and len(output):
```
Comment (ryanpetrello) [inline awxkit/awxkit/cli/format.py:149]: image

File: awxkit/awxkit/cli/resource.py (hunk 1 above)
Comment (kdelee) [inline awxkit/awxkit/cli/resource.py:14]: :+1:

**Review by kdelee — APPROVED** [review]
:shipit:

File: awxkit/awxkit/cli/resource.py (hunk 1 above)
Comment (kdelee) [reply awxkit/awxkit/cli/resource.py:14]: @squidboylan

**Review by squidboylan — APPROVED** [review]
:D

**Review by jbradberry — APPROVED** [review]
Seems good

**Pull request merged**